docs/getting_started/installation.rst

```diff
@@ -72,6 +72,9 @@ line in the `app/AppKernel.php` file of your project:
 
                 // And finally, the storage and SonataAdminBundle
                 new Sonata\AdminBundle\SonataAdminBundle(),
+                
+                // For SonataDoctrineORMAdmin add
```

Comment:
IMO if we add this we should also add the other persistence bundles aswell

Reply:
I dont't think so, because we have:
`composer require sonata-project/doctrine-orm-admin-bundle` 
in point 
https://sonata-project.org/bundles/admin/3-x/doc/getting_started/installation.html#download-a-storage-bundle 
which is't included in the next step